modules/local/ragtag/main.nf

```diff
@@ -10,39 +10,40 @@ process RAGTAG_SCAFFOLD {
     tuple val(meta), path(assembly), path(reference)
 
     output:
-    tuple val(meta), path("${assembly}_ragtag_${reference}/*.fasta"), emit: corrected_assembly
-    tuple val(meta), path("${assembly}_ragtag_${reference}/*.agp"), emit: corrected_agp
-    tuple val(meta), path("${assembly}_ragtag_${reference}/*.stats"), emit: corrected_stats
+    tuple val(meta), path("*.fasta"), emit: corrected_assembly
+    tuple val(meta), path("*.agp"), emit: corrected_agp
+    tuple val(meta), path("*.stats"), emit: corrected_stats
```

Comment:
this could be aligned, it makes it easier to read
```suggestion
    tuple val(meta), path("*.fasta"), emit: corrected_assembly
    tuple val(meta), path("*.agp"),   emit: corrected_agp
    tuple val(meta), path("*.stats"), emit: corrected_stats
```